core/variant/variant.h

```diff
@@ -836,6 +836,19 @@ struct StringLikeVariantComparator {
 	static bool compare(const Variant &p_lhs, const Variant &p_rhs);
 };
 
+struct StringLikeVariantOrder {
+	static _ALWAYS_INLINE_ bool compare(const Variant &p_lhs, const Variant &p_rhs) {
```

Comment:
This static method isn't technically needed with the alternative solution (or when using `List<Variant>` in the visual shader code) but I think it's useful to keep so we can use it when sorting `Array` as it takes a `Callable` and we can use `callable_mp_static`

I will look at adding such static methods to other comparators to aid in using them for such cases